config/zephyr/Kconfig

```diff
@@ -477,7 +477,19 @@ config CHIP_FACTORY_RESET_ERASE_NVS
 	  regain the original storage performance if any firmware issue has brought
 	  it to an unexpected state. For this reason, set this option if the entire
 	  configuration is supposed to be cleared on a factory reset, including
-	  device-specific entries.
+	  device-specific entries. It makes sense only for NVS filesystem.
+
+config CHIP_FACTORY_RESET_ERASE_ZMS
+	bool "Erase ZMS flash pages on factory reset"
+	depends on SETTINGS_ZMS
```

Comment:
Do we have protection on setting this config when DAC migration is turned on? I mean we should not allow for setting this config if DAC migration is enabled.

Reply:
Yes I've added it to the nrfconnect's .Kconfig file, but please double check it :)